### ISSUE TITLE
Add padding right when Background is active

### ISSUE THREAD

**Issue by LukasPolak** [aaron-bond/better-comments#125]
Hi!
Is there any option to add some padding right (or some spacing utility), when I use a background highlighting.
**Problem:** 
Last character of comment is cutted out without background

It's looking like this:
![original](https://user-images.githubusercontent.com/17145302/48670123-a2ea4180-eb12-11e8-939c-313ef060554b.png)

It's should looks like this:
![wanted](https://user-images.githubusercontent.com/17145302/48670124-a2ea4180-eb12-11e8-8c5e-fa8d1995b504.png)
Thanks a lot!

**Comment by aaron-bond** [comment]
Good idea. I think we should only do this when the background color property is active though, so that we don't mess up any existing scenarios. Does that sound OK?

**Comment by LukasPolak** [comment]
That sounds great! Thanks.
